src/v2/Components/Authentication/Views/SignUpForm.tsx

```diff
@@ -305,6 +288,7 @@ export class SignUpForm extends Component<SignUpFormProps, SignUpFormState> {
                   showRecaptchaDisclaimer={this.props.showRecaptchaDisclaimer}
                 />
               </Join>
+              <MntnConversionPixel path="/signup" />
```

Comment:
Ditto, no need to pass path

Reply:
In this case there is no path for the component rendering the pixel, so it has to be passed. This is the case where we need to track a conversion event inside the signup modal.

Reply:
Would also be useful to use in the case of gallery inquiries but a double render there is removing the pixel immediately after it's added to the DOM. May work on this in a follow up.